app/views/message_threads/bulk/exports/edit.html.erb

```diff
@@ -1,61 +1,74 @@
+            <%= check_box_tag "export[settings][summary]", "1", @export.settings["summary"], class: "h-5 w-5 rounded border-gray-300 text-blue-600 focus:ring-blue-600 cursor-pointer" %>
+            <%= label_tag "export[settings][summary]", "Exportovať sumár", class: "select-none text-sm/6 text-gray-900 cursor-pointer" %>
           </div>
 
-          <table class="min-w-full">
-            <tbody>
-            <tr class="border-t border-gray-200">
-              <td colspan="3" class="border-t border-gray-200 bg-gray-50 py-2 pl-4 pr-3 text-left text-sm font-semibold text-gray-900 sm:pl-3">Všeobecné nastavenie</td>
-            </tr>
-            <tr class="border-t border-gray-200">
-              <td class="py-2 pr-3"><%= check_box_tag "export[settings][default]", "1", @export.settings['default'], class: "h-5 w-5 rounded border-gray-300 text-blue-600 focus:ring-blue-600 cursor-pointer" %></td>
-              <td class="text-sm">všetky správy</td>
-              <td class="whitespace-nowrap w-full py-4 pl-3 pr-4 text-right text-sm font-medium sm:pr-3"><%= text_field_tag "export[settings][templates][default]", @export.settings.dig('templates', 'default'), class: "block w-full rounded-md bg-white px-3 py-1.5 text-base text-gray-900 outline outline-1 -outline-offset-1 outline-gray-300 placeholder:text-gray-400 focus:outline focus:outline-2 focus:-outline-offset-2 focus:outline-indigo-600 sm:text-sm/6" %></td>
-            </tr>
-            <tr>
-              <td colspan="3" class="border-t border-gray-200 bg-gray-50 py-2 pl-4 pr-3 text-left text-sm font-semibold text-gray-900 sm:pl-3">Podľa typu formulára</td>
-            </tr>
-            <% @message_forms.each do |message_form| %>
-              <tr class="border-t border-gray-200">
-                <td><%= check_box_tag "export[settings][by_form][#{message_form}]", "1", @export.settings.dig('by_form', message_form), class: "h-5 w-5 rounded border-gray-300 text-blue-600 focus:ring-blue-600 cursor-pointer" %></td>
-                <td class="text-sm"><%= message_form %></td>
-                <td class="whitespace-nowrap py-4 pl-3 pr-4 text-right text-sm font-medium sm:pr-3">
-                  <%= text_field_tag "export[settings][templates][#{message_form}]", @export.settings.dig('templates', message_form), placeholder: 'nastavenie ako pre všetky správy', class: "block w-full rounded-md bg-white px-3 py-1.5 text-base text-gray-900 outline outline-1 -outline-offset-1 outline-gray-300 placeholder:text-gray-400 focus:outline focus:outline-2 focus:-outline-offset-2 focus:outline-indigo-600 sm:text-sm/6" %>
-                </td>
-              </tr>
-            <% end %>
+          <div class="flex items-center gap-x-3 py-3">
+            <%= check_box_tag "export[settings][messages]", "1", @export.settings["messages"], class: "h-5 w-5 rounded border-gray-300 text-blue-600 focus:ring-blue-600 cursor-pointer", onclick: "this.form.requestSubmit()" %>
```

Comment:
neviem ci to uplne treba komplikovat ale aj taketo by sa dalo https://www.stimulus-components.com/docs/stimulus-auto-submit